src/main/java/com/jmb/springfactory/LoggerAop.java

```diff
@@ -0,0 +1,40 @@
+package com.jmb.springfactory;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.annotation.Pointcut;
+import org.jboss.logging.Logger;
+
+@Aspect
+public class LoggerAop {
+  
+  private static final String LOG_PARAM = "with args: %s";
+  private static final String LOG_EXECUTING_METHOD = "Executing method %s";
+  private static final String LOG_FINALIZING_METHOD = "Finalizing method %s";
+  private Logger monitorAopLog = Logger.getLogger(getClass());
+
+  @Pointcut("execution(* com.jmb.springfactory.*.*(..))")
+  public void allMethods() {}
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)